Library/Homebrew/rubocops/components_order.rb

```diff
@@ -123,8 +131,20 @@ def check_on_os_block_content(component_precedence_list, on_os_block)
 
             valid_node ||= on_os_allowed_methods.include? child.method_name.to_s
 
+            shadowed = false
+            if child.method_name.to_s == "keg_only"
+              child.arguments.each do |arg|
```

Comment:
```suggestion
              shadowed = child.arguments.find do |arg|
```

Reply:
I _think_ it should work and be simpler.

Reply:
It works